test/e2e/examples1_test.go

```diff
@@ -170,6 +173,17 @@ func (suite *ExamplesTestSuite) TestBusinessApp() {
 	require.NoError(t, err, "SmokeTest failed")
 }
 
+func getBusinessAppCR(err error) *os.File {
+	content, err := ioutil.ReadFile("../../examples/business-application-injected-sidecar.yaml")
+	require.NoError(t, err)
+	newContent := strings.Replace(string(content), "image: jaegertracing/vertx-create-span:operator-e2e-tests", "image: "+vertxExampleImage, 1)
```

Comment:
Sounds like a hack to me (YAML file -> to string -> replace -> to file -> use YAML in request to Kubernetes), but I'm happy if you are :-)

Reply:
Well, it was done on a Monday morning.  :-)  Let me take a look and see how much work it is to do this the right way.